src/fpnn/model_training.py

```diff
@@ -1,15 +1,18 @@
+from pathlib import Path
+from typing import Sequence
 import matplotlib.pyplot as plt
 import numpy as np
 import pickle as pkl
 
 from sklearn.metrics import r2_score
 from sklearn.preprocessing import StandardScaler
-from tensorflow import keras
+from tensorflow import keras, random, data
 from keras import layers, models, utils
 
 
 # Set a random seed
 utils.set_random_seed(42)
+random.set_seed(42)
```

Comment:
- [ ] Define a var for the seed. Useful when spreading the same seed value for different seed setters.